pkg/cluster/cluster.go

```diff
@@ -370,7 +370,10 @@ func (c *Cluster) delete() {
 		}
 	}
 	if c.spec.Backup != nil {
-		k8sutil.DeleteBackupReplicaSetAndService(c.KubeCli, c.Name, c.Namespace, c.spec.Backup.CleanupBackupsOnClusterDelete)
+		err := c.bm.CleanupBackups()
+		if err != nil {
+			panic(err)
```

Comment:
We should change this func to return err and log outside. Maybe in a separate PR?